### PR TITLE
JAMES-2178 Restart cassandra after 25 tests in CassandraMessageMapper…

### PR DESCRIPTION
**Pull request by chibenwa**
…Test

### CODE REVIEW

File: mailbox/cassandra/src/test/java/org/apache/james/mailbox/cassandra/mail/CassandraMessageMapperTest.java [hunk 1]
```diff
@@ -40,11 +41,21 @@
...
+import org.junit.Rule;
+import org.junit.rules.TestRule;
 
 public class CassandraMessageMapperTest extends MessageMapperTest {
     
     @ClassRule public static DockerCassandraRule cassandraServer = new DockerCassandraRule();
-    
+
+    public static ContainerLifecycleConfiguration cassandraLifecycleConfiguration = ContainerLifecycleConfiguration.builder()
+        .iterationsBetweenRestart(25)
```
Comment (aduprat) [inline mailbox/cassandra/src/test/java/org/apache/james/mailbox/cassandra/mail/CassandraMessageMapperTest.java]: In _CassandraGetMessageListMethodTest_ it's set to 20, can we use the same value everywhere (constant)?

Comment (aduprat) [reply mailbox/cassandra/src/test/java/org/apache/james/mailbox/cassandra/mail/CassandraMessageMapperTest.java]: What is the main reason of that? Compaction or something like that?

Comment (chibenwa) [reply mailbox/cassandra/src/test/java/org/apache/james/mailbox/cassandra/mail/CassandraMessageMapperTest.java]: Well, if you want

**Comment by chibenwa** [comment]
test this please

**Comment by rouazana** [comment]
you then need to change your commit title :)

**Comment by chibenwa** [comment]
:-(

Will  also reschedule the build on CI

**Comment by chibenwa** [comment]
I'll rename the commit when merging (save CI time).

@rouazana  ?

**Comment by chibenwa** [comment]
```
[ERROR] Errors: 
[ERROR]   CassandraSetMessagesMethodTest>SetMessagesMethodTest.attachmentsShouldBeRetrievedWhenChainingSetMessagesAndGetMessagesTextAttachment:2985 » ConditionTimeout
```

File: mailbox/cassandra/src/test/java/org/apache/james/mailbox/cassandra/mail/CassandraMessageMapperTest.java [hunk 2]
```diff
@@ -40,11 +41,21 @@
...
 import org.junit.Before;
 import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.rules.TestRule;
 
 public class CassandraMessageMapperTest extends MessageMapperTest {
     
     @ClassRule public static DockerCassandraRule cassandraServer = new DockerCassandraRule();
-    
+
```
Comment (strokyl) [inline mailbox/cassandra/src/test/java/org/apache/james/mailbox/cassandra/mail/CassandraMessageMapperTest.java]: I don't understand I thought container where restarted between each test ?

Comment (mbaechler) [reply mailbox/cassandra/src/test/java/org/apache/james/mailbox/cassandra/mail/CassandraMessageMapperTest.java]: it's not, we create a new database for each test, isolation is supposed to be enough

**Pull request merged**